src/services/babel-browser.gen.js

```diff
@@ -0,0 +1,78 @@
+const babel = require('@babel/core');
+const presetEnv = require('@babel/preset-env');
+
+const browserList = require('../../config/browsers.json');
+
+module.exports = () => {
+  const browserListForBabel = {};
+
+  for (const browser in browserList) {
+    if (browser !== 'chromium') {
+      browserListForBabel[browser] = browserList[browser];
+    }
+  }
+
+  // preset-env doesn't give an introspection API, so we have to get it from
+  // the metadata on transformed code
```

Comment:
Can we self-document instead of commenting? E.g. wrap this in a function called `extractPluginsFromNoopCompilationResult` or something? (Extracting a function or two would probably be good from a composition standpoint anyway)